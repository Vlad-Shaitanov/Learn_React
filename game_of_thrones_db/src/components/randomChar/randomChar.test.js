import React from "react";
import RandomChar from "./randomChar.js";
import renderer from "react-test-renderer";

describe("Testing <RandomChar/>", () => {//test suit
	//test case
	it("RandomChar have rendered correctly", () => {
		//В переменной экземпляр тестируемого компонента
		const char = renderer.create(<RandomChar />).toJSON();
		/*После запуска теста будет сделан снимок нашего компонента. В дальнейшем
		этот снимок будет сравниваться с оригиналом, те. полученным в начале*/
		expect(char).toMatchSnapshot();
	});

});