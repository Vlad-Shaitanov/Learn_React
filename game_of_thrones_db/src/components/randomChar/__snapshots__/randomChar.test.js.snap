// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <RandomChar/> RandomChar have rendered correctly 1`] = `
<div
  className="random-block rounded"
>
  <div
    class="loadingio-spinner-spin-ujjn0oqt6tf"
  >
    <div
      class="ldio-asfxmtcg4i"
    >
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
    </div>
  </div>
</div>
`;
